package.json

```diff
@@ -48,10 +48,10 @@
 		"@types/xmlrpc": "^1.3.6",
 		"@typescript-eslint/eslint-plugin": "^5.8.1",
 		"@typescript-eslint/parser": "^5.8.1",
-		"eslint": "^8.5.0",
+		"eslint": "^8.49.0",
```

Comment:
can revert these, also a result of prettier. please advice

Reply:
weird, yeah can we revert package and package-lock?